src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  let result = rent * days;
+
+  if (days >= 7) {
+    result -= 50;
```

Comment:
Use a constant with a descriptive name instead of the literal `7` to represent the threshold for a long term rental discount, and another constant for the `50` to represent the discount amount, as per the checklist.